### ISSUE TITLE
<main> tag - span across multiple sections

### ISSUE THREAD

**Issue by besweeet** [elementor/elementor#13441]
## Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
Per WCAG guidelines 1.3.1, sections need **header**, **footer**, and **main** tags. The first two are covered in recent Elementor updates (yay!). But, the **main** tag is the issue.

While you can set a **main** tag on a per-section basis, pages should only have one **main** tag. If I set **main** for all the sections, there will be multiple tags, which isn't the right way of doing it.

Basically, everything that falls between **header** and **footer** should be bundled within **main**.

**Comment by besweeet** [comment]
Bump. I was hoping this would be an easy thing to solve for. Basically, wrap all of a page's sections in a <main> tag.

**Comment by superterrific** [comment]
I have this issue as well.

**Comment by rami-elementor** [comment]
When you create a new Header, Footer, or any other **Theme Builder** template, you can click the settings icon at the bottom of the Editor panel. This will open the template settings page. There you will find the option to select the HTML tag for that template.

![Elementor-Site-Header](https://user-images.githubusercontent.com/92088692/215733828-14b26d25-812e-40f3-857d-2d444e743cc2.png)

For header templates you can choose the `<header>` tags.
For footer templates you can choose the `<footer>` tags.
For all other pages you can choose the `<main>` tags.
